packages/ui-extensions-dev-console/src/sections/Extensions/components/PostPurchaseModal/PostPurchaseModal.tsx

```diff
@@ -0,0 +1,64 @@
+import en from './translations/en.json'
+import * as styles from './PostPurchaseModal.module.scss'
+import React from 'react'
+import {Modal, ModalProps} from '@shopify/polaris'
+import {useI18n} from '@shopify/react-i18n'
+import {ClipboardMinor} from '@shopify/polaris-icons'
+import {toast} from 'react-toastify'
+import {IconButton} from '@/components/IconButton'
+
+interface Props extends Pick<ModalProps, 'open' | 'onClose'> {
+  url?: string
+}
+
+export function PostPurchaseModal({url, ...modalProps}: Props) {
+  const [i18n] = useI18n({
+    id: 'PostPurchaseModal',
+    fallback: en,
+  })
+
+  function handleCopyPreviewLink() {
+    if (!url) {
+      return
+    }
+
+    navigator.clipboard
+      .writeText(url)
+      .then(() => {
+        toast(i18n.translate('copy.success'), {toastId: `copy-${url}`})
+      })
+      .catch(() => {
+        toast(i18n.translate('copy.error'), {type: 'error', toastId: `copy-${url}-error`})
+      })
+  }
+
+  return (
+    <Modal title={i18n.translate('title')} {...modalProps}>
+      <div className={styles.Content}>
+        <ol className={styles.Instructions}>
+          <li>
+            {i18n.translate('point1.intro')}{' '}
+            <a href="" target="_blank">
```

Comment:
Is the plan to add this link later on?

Reply:
Oops! Thanks for catching that.  Added the URL and some unit tests